notebooks/Model_Output/Downloading model fields with NCSS.ipynb

```diff
@@ -351,7 +376,7 @@
     "              transform=data_projection, zorder=0)\n",
     "ax.contour(longitude, latitude, height_var[time_index].squeeze(), contours, colors='k',\n",
     "           transform=data_projection, linewidths=2, zorder=1)\n",
-    "ax.set_title(ds[time_name][time_index].values)\n",
+    "ax.set_title(temp_var.time[time_index].values)\n",
```

Comment:
Is `temp_var.time` guaranteed to work? I thought this had to go through the `.metpy` accessor somewhere to find the one pulled out based on metadata.